app/src/androidTest/java/com/example/pygmyhippo/admin/AllEventsFragmentTest.java

```diff
@@ -79,35 +79,36 @@ public void setup() {
-        final Context context = InstrumentationRegistry.getInstrumentation().getTargetContext();
-        final String[] userCategories = context.getResources().getStringArray(R.array.all_events_category_spinner);
-
-        for (String category : userCategories) {
-            onView(withId(R.id.a_alllist_category_spinner)).perform(click());
-            onData(allOf(is(instanceOf(CharSequence.class)), is(category))).perform(click());
-            onView(withText(category)).check(ViewAssertions.matches(isDisplayed()));
-        }
-    }
-
-    @Test
-    public void testOrderSpinner() {
-        final Context context = InstrumentationRegistry.getInstrumentation().getTargetContext();
-        final String[] spinnerOrders = context.getResources().getStringArray(R.array.order_spinner);
-
-        for (String order : spinnerOrders) {
-            onView(withId(R.id.a_alllist_order_spinner)).perform(click());
-            onData(allOf(is(instanceOf(CharSequence.class)), is(order))).perform(click());
-            onView(withText(order)).check(ViewAssertions.matches(isDisplayed()));
-        }
-    }
-
-    @Test
-    public void testAllEventList() {
-        // TODO: Add more thorough testing for items in all list after navigation is figured out.
-        onView(withId(R.id.a_alllist_recycler))
-                .perform(RecyclerViewActions.actionOnItemAtPosition(0, click()));
-    }
+// These tests done run since I deleted teh spinners
+//    @Test
+//    public void testEventCategorySpinner() {
+//        final Context context = InstrumentationRegistry.getInstrumentation().getTargetContext();
+//        final String[] userCategories = context.getResources().getStringArray(R.array.all_events_category_spinner);
+//
+//        for (String category : userCategories) {
+//            onView(withId(R.id.a_alllist_category_spinner)).perform(click());
+//            onData(allOf(is(instanceOf(CharSequence.class)), is(category))).perform(click());
+//            onView(withText(category)).check(ViewAssertions.matches(isDisplayed()));
+//        }
+//    }
+//
+//
+//    @Test
+//    public void testOrderSpinner() {
+//        final Context context = InstrumentationRegistry.getInstrumentation().getTargetContext();
+//        final String[] spinnerOrders = context.getResources().getStringArray(R.array.order_spinner);
+//
+//        for (String order : spinnerOrders) {
+//            onView(withId(R.id.a_alllist_order_spinner)).perform(click());
+//            onData(allOf(is(instanceOf(CharSequence.class)), is(order))).perform(click());
+//            onView(withText(order)).check(ViewAssertions.matches(isDisplayed()));
+//        }
+//    }
+//
+//    @Test
+//    public void testAllEventList() {
+//        // TODO: Add more thorough testing for items in all list after navigation is figured out.
+//        onView(withId(R.id.a_alllist_recycler))
+//                .perform(RecyclerViewActions.actionOnItemAtPosition(0, click()));
+//    }
```

Comment:
How much testing is left for this fragment? Do we test if stuff like the title displays at least?

Reply:
I have not idea, I honestly think it's good. there is no title on this fragment.

Reply:
oh wait i miss read this, ill make one test lol